assets/package.json

```diff
@@ -3,17 +3,18 @@
   "version": "0.1.0dev",
   "description": "National Ground Water Monitoring Network",
   "scripts": {
-    "test": "karma start",
-    "test:chrome": "karma start --browsers Chrome",
-    "test:firefox": "karma start --browsers Firefox",
-    "test:watch": "karma start --debug --no-single-run --auto-watch --browsers Chrome",
+    "test": "mkdir -p dist/scripts && run-p build:vendorjs && karma start",
+    "test:chrome": "mkdir -p dist/scripts && run-p build:vendorjs && karma start --browsers Chrome",
+    "test:firefox": "mkdir -p dist/scripts && run-p build:vendorjs &&karma start --browsers Firefox",
+    "test:watch": "mkdir -p dist/scripts && run-p build:vendorjs && karma start --debug --no-single-run --auto-watch --browsers Chrome",
     "clean": "rm -rf dist && mkdir dist",
     "build": "npm run clean && run-p build:* && npm run hash-assets",
     "build:autotrack": "uglifyjs --output dist/autotrack.js node_modules/autotrack/autotrack.js",
     "build:css": "node-sass --include-path node_modules/uswds/dist/scss src/styles/main.scss dist/main.css && uglifycss --output dist/main.css dist/main.css && postcss dist/main.css --no-map --use autoprefixer postcss-flexbugs-fixes -o dist/main.css",
     "build:fonts": "mkdir -p dist/fonts && cp -r node_modules/uswds/dist/fonts/* dist/fonts && cp node_modules/@fortawesome/fontawesome-free/webfonts/* dist/fonts",
     "build:images": "mkdir -p dist/img && mkdir -p dist/img && cp -r src/images/* dist/img && cp -r node_modules/uswds/dist/img/* dist/img",
     "build:patterns": "mkdir -p dist/patterns && cp src/patterns/* dist/patterns",
+    "build:vendorjs": "mkdir -p dist/scripts && concat -o dist/scripts/vendor.js node_modules/list.js/dist/list.js",
```

Comment:
Why did we need to do this - I'd think the rollup-plugin-common-js would support importing this directly?

Reply:
The watch task needs to call this too.

Reply:
It didn't work for me. I tried to use ```import { List } from 'list.js';`` Is there some configuration I need to add to commonjs().

Reply:
Might be a default export? `import List from 'list.js'`

Reply:
So tried it again and actually looked at the error message. I need to do ```import { default as List } from 'list.js';```. And then it works. I will remove the vendor.js stuff.

Reply:
Could have sworn I tried what you are suggesting too but maybe I had more than one problem in the build at the time!